tests/unit/memcache_api_test.py

```diff
@@ -14,5 +13,4 @@ def setUp(self):
         self.setUpCache(memcache_api.MemcacheApi())
 
     def tearDown(self):
```

Comment:
Can we just delete this file?

Reply:
Yes. I started this PR before we talked about switching over to datastore cache usage at the same time as the rest of python3 feature branch. Now there's no need to keep both. Will follow up in another PR.